.tool-versions

```diff
@@ -0,0 +1 @@
+elixir 1.16.3
```

Comment:
This was accidentally commited?

Reply:
yes :) When testing, there appear to be errors on 1.17 so I did this to pin it to 1.16.3 🥶 could very easily have been a false positive or something dumb I was doing.